### PR TITLE
i added the play game view

### PR DESCRIPTION
**Pull request by mnour98**
please check the code

### CODE REVIEW

File: src/View/PlayGameView.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
+package View;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class PlayGameView extends ParentView  implements ActionListener {
```
Comment (mnour98) [inline src/View/PlayGameView.java:8]: why is this here

**Comment by ghazar24** [comment]
everything is okay
